src/main/java/com/example/account/aggregate/AccountAggregate.java

```diff
@@ -1,36 +1,18 @@
 package com.example.account.aggregate;
 
-import com.example.account.command.CreateAccountCommand;
-import com.example.account.event.AccountCreatedEvent;
 import com.example.account.event.BidStatus;
-import com.example.account.event.Event;
-import com.example.account.repository.EventRepository;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import lombok.AllArgsConstructor;
-import lombok.SneakyThrows;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.stereotype.Service;
-import reactor.core.publisher.Mono;
+import lombok.Data;
+import org.springframework.data.relational.core.mapping.Table;
 
-@Slf4j
-@Service
-@AllArgsConstructor
-public class AccountAggregate {
+@Data
+@Table("event")
```

Comment:
why aggregate's table "event"?

Reply:
I want to persist events in the database( or I have to save aggregates to the database instead of events? Or I should have 2 databases for aggregates and events. Kill me, please....

Reply:
@amerkul you should have an `events` table with all the events AND the `aggregate` table with aggregate